src/main/java/lambdastreams/task3/PersonalData.java

```diff
@@ -0,0 +1,23 @@
+package lambdastreams.task3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PersonalData {
+    public static void main(String[] args) {
```

Comment:
the same here - just convert it to the test (using @Test annotation), write some assertion and move it to the test package